### PR TITLE
Maia

### PR DESCRIPTION
**Pull request by jobrs**
Ready for staging deployment:
* Add Maia deployment
* Point maia-ingress to maia pod
* Add missing service for Maia-Prometheus
* Add Maia config and policy file
* Seed Maia technical user and roles

### CODE REVIEW

File: openstack/maia/templates/seed.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  requires:
+#    - keystone-seed
+#    - monsoon3/domain-cc3test-seed
+#    - monsoon3/domain-ccadmin-seed
+
+  roles:
+  - monitoring_viewer
+  - monitoring_admin
+
+# TODO expose as OpenStack service
```
Comment (auhlig) [inline openstack/maia/templates/seed.yaml]: Is it really intended to expose maia as an OpenStack service? I thought it's just going to be a "wrapper" for customer's prometheus (see [here](https://github.com/sapcc/maia#using-maia)) and Grafana.

File: openstack/maia/templates/seed.yaml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+      - group: CCADMIN_MONITORING_USERS
+        role: monitoring_viewer
+
+  # TODO: reactivate once we have tests
+  # - name: cc3test
+  #   projects:
+  #   - name: monitoring_test
+  #     description: 'project for used for testing the monitoring service'
+  #   users:
+  #   - name: test_monasca_agent
```
Comment (auhlig) [inline openstack/maia/templates/seed.yaml]: monasca?

File: openstack/maia/templates/seed.yaml (hunk 1 above)
Comment (jobrs) [inline openstack/maia/templates/seed.yaml]: yes, for consumption by Elektra. Not today, but tomorrow --> TODO

The roles are needed today, therefore they are not commented out

File: openstack/maia/templates/seed.yaml (hunk 2 above)
Comment (jobrs) [inline openstack/maia/templates/seed.yaml]: no ;-) , the customer monitoring service implemented by Maia

File: openstack/maia/templates/seed.yaml (hunk 1 above)
Comment (auhlig) [inline openstack/maia/templates/seed.yaml]: Ah. I wasn't aware this is planned. Thanks.

**Comment by auhlig** [comment]
Go maia!

**Pull request merged**